Charter.md

```diff
@@ -11,7 +11,7 @@ also easily visible to outsiders.
 
 Express is a http web server framework with a simple and expressive API
 which is highly aligned with Node.js core. We aim to be the best in
-class for writing performant, spec compliant, and powerful web servers
+class for writing preformat, spec compliant, and powerful web servers
```

Comment:
The word `preformat` is different from `performant` and has a different meaning. The current word here is correct: https://www.google.com/search?q=define+performant